platform/ios/src/MLNMapView.h

```diff
@@ -1364,6 +1364,18 @@ MLN_EXPORT
  */
 - (CGPoint)anchorPointForGesture:(UIGestureRecognizer *)gesture;
 
+
+/**
+ * Sets a LatLngBounds that constraints map transformations to this bounds.
+ * @param latLngBounds the bounds to constrain the map with
+ */
+- (void)setLatLngBounds:(MLNCoordinateBounds)latLngBounds;
```

Comment:
Can it be called something other than `setLatLngBounds`? The iOS/macOS SDK now has a method named after the Android SDK’s `LatLngBounds` class, which makes little sense. That class is analogous to the `MGLCoordinateBounds` struct on iOS/macOS, so it should be called something like `setCoordinateBounds`. Better yet, it should be modeled as a property called something more descriptive like `viewableCoordinateBounds`. As it is, this is a write-only property, nearly unprecedented in any Objective-C framework.